src/app/zap-templates/zcl/data-model/chip/laundry-washer-cluster.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="SemanticTags" type="ENUM16">
+    <cluster code="0x0051"/>
+    <item value="0x4000" name="Normal"/>
+    <item value="0x4001" name="Delicate"/>
+    <item value="0x4002" name="Heavy"/>
+    <item value="0x4003" name="Whites"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Laundry Washer</name>
```

Comment:
The spec name is Laundry Washer Mode Select, no?

Reply:
This has changed since I was working on it. I was working of a locally built copy. I will change it to match the current spec.